system_test/aest_hard_fork_SUITE.erl

```diff
@@ -0,0 +1,334 @@
+-module(aest_hard_fork_SUITE).
+
+%=== EXPORTS ===================================================================
+
+% Common Test exports
+-export([all/0]).
+-export([groups/0]).
+-export([suite/0]).
+-export([init_per_suite/1]).
+-export([end_per_suite/1]).
+-export([init_per_group/2]).
+-export([end_per_group/2]).
+-export([init_per_testcase/2]).
+-export([end_per_testcase/2]).
+
+% Test cases
+-export([
+         node_persisting_chain_and_not_mining_has_genesis_as_top/1,
```

Comment:
I'd prefer the test case name to explicitly show which node is concerned in the test. In this case it's `old_node_persisting_chain_and_not_mining_has_genesis_as_top`.

Reply:
Addressed in d641ba52